### PR TITLE
bqexecute new tests

### PR DESCRIPTION
**Pull request by priyabhatnagar25**

### CODE REVIEW

File: src/e2e-test/java/io/cdap/plugin/bigqueryexecute/runner/TestRunner.java [hunk 1]
```diff
@@ -25,7 +25,8 @@
 @RunWith(Cucumber.class)
 @CucumberOptions(
   features = {"src/e2e-test/features"},
-  glue = {"io.cdap.plugin.bigqueryexecute.stepsdesign", "io.cdap.plugin.common.stepsdesign", "stepsdesign"},
+  glue = {"io.cdap.plugin.bigqueryexecute.stepsdesign", "io.cdap.plugin.common.stepsdesign", "stepsdesign",
+    "io.cdap.plugin.common.stepsdesign"},
```
Comment (itsmekumari) [inline src/e2e-test/java/io/cdap/plugin/bigqueryexecute/runner/TestRunner.java]: Why is this change needed? Please revert as it is repeated.

File: src/e2e-test/java/io/cdap/plugin/bigqueryexecute/runner/TestRunnerRequired.java [hunk 2]
```diff
@@ -25,7 +25,8 @@
 @RunWith(Cucumber.class)
 @CucumberOptions(
   features = {"src/e2e-test/features"},
-  glue = {"io.cdap.plugin.bigqueryexecute.stepsdesign", "io.cdap.plugin.common.stepsdesign", "stepsdesign"},
+  glue = {"io.cdap.plugin.bigqueryexecute.stepsdesign", "io.cdap.plugin.common.stepsdesign", "stepsdesign",
+  "io.cdap.plugin.common.stepsdesign"},
```
Comment (itsmekumari) [inline src/e2e-test/java/io/cdap/plugin/bigqueryexecute/runner/TestRunnerRequired.java]: Remove the change done here as well, as this is repeated.

File: src/e2e-test/features/bigqueryexecute/BQExecute.feature [hunk 3]
```diff
@@ -173,3 +173,44 @@ Feature: BigQueryExecute - Verify data transfer using BigQuery Execute plugin
...
     Then Verify 1 records updated in BigQuery table: "bqSourceTable" with query "bqExecuteCountDMLUpdate"
+
+  @BQEXECUTE_SOURCE_TEST @BQ_SINK_TEST @BQ_EXECUTE_SQL @BQExecute_Required
+  Scenario: Verify the successful records transfer from Bigquery(source) to Bigquery(sink) using BQExecute with allow relaxation schema as false.
+    Given Open Datafusion Project to configure pipeline
+    When Expand Plugin group in the LHS plugins list: "Conditions and Actions"
+    When Select plugin: "BigQuery Execute" from the plugins list as: "Conditions and Actions"
+    When Navigate to the properties page of plugin: "BigQuery Execute"
+    Then Replace input plugin property: "projectId" with value: "projectId"
+    Then Enter textarea plugin property: "sql" with value: "bqExecuteQuery"
```
Comment (itsmekumari) [inline src/e2e-test/features/bigqueryexecute/BQExecute.feature]: Don't use select query, instead use the Insert sql query to insert the record through BQ execute. Refer ITN class.

File: src/e2e-test/features/bigqueryexecute/BQExecute.feature [hunk 4]
```diff
@@ -173,3 +173,44 @@ Feature: BigQueryExecute - Verify data transfer using BigQuery Execute plugin
     Then Open and capture logs
     Then Verify the pipeline status is "Succeeded"
     Then Verify 1 records updated in BigQuery table: "bqSourceTable" with query "bqExecuteCountDMLUpdate"
+
+  @BQEXECUTE_SOURCE_TEST @BQ_SINK_TEST @BQ_EXECUTE_SQL @BQExecute_Required
+  Scenario: Verify the successful records transfer from Bigquery(source) to Bigquery(sink) using BQExecute with allow relaxation schema as false.
```
Comment (itsmekumari) [inline src/e2e-test/features/bigqueryexecute/BQExecute.feature]: Change to: Verify the successful records transfer from Bigquery source to Bigquery sink using insert sql query in BQExecute with rowAsArguments as false

File: src/e2e-test/java/io/cdap/plugin/bigqueryexecute/runner/TestRunner.java (hunk 1 above)
Comment (priyabhatnagar25) [reply src/e2e-test/java/io/cdap/plugin/bigqueryexecute/runner/TestRunner.java]: reverted

File: src/e2e-test/java/io/cdap/plugin/bigqueryexecute/runner/TestRunnerRequired.java (hunk 2 above)
Comment (priyabhatnagar25) [reply src/e2e-test/java/io/cdap/plugin/bigqueryexecute/runner/TestRunnerRequired.java]: reverted

File: src/e2e-test/features/bigqueryexecute/BQExecute.feature (hunk 3 above)
Comment (priyabhatnagar25) [reply src/e2e-test/features/bigqueryexecute/BQExecute.feature]: insert query used

File: src/e2e-test/features/bigqueryexecute/BQExecute.feature (hunk 4 above)
Comment (priyabhatnagar25) [reply src/e2e-test/features/bigqueryexecute/BQExecute.feature]: changed

File: src/e2e-test/java/io/cdap/plugin/bigqueryexecute/runner/TestRunner.java [hunk 5]
```diff
@@ -25,7 +25,8 @@
 @RunWith(Cucumber.class)
 @CucumberOptions(
   features = {"src/e2e-test/features"},
-  glue = {"io.cdap.plugin.bigqueryexecute.stepsdesign", "io.cdap.plugin.common.stepsdesign", "stepsdesign"},
+  glue = {"io.cdap.plugin.bigqueryexecute.stepsdesign", "io.cdap.plugin.common.stepsdesign", "stepsdesign",
+    "io.cdap.plugin.bigquery.stepsdesign"},
```
Comment (itsmekumari) [inline src/e2e-test/java/io/cdap/plugin/bigqueryexecute/runner/TestRunner.java:29]: Why is the big query related step design added?

Comment (priyabhatnagar25) [reply src/e2e-test/java/io/cdap/plugin/bigqueryexecute/runner/TestRunner.java:29]: This is required because we are doing validation step for the above scenario, validation for the content of records transferred

File: src/e2e-test/resources/pluginParameters.properties [hunk 6]
```diff
@@ -318,6 +318,7 @@ bqeSourceOutputSchema={ "type": "record", "name": "text", "fields": [ \
 bqExecuteDDLCreate=CREATE TABLE `PROJECT_NAME.DATASET.TABLENAME` (transaction_info BOOL, transaction_num BYTES, transaction_uid STRING)
 bqExecuteDDLDrop=DROP TABLE `PROJECT_NAME.DATASET.TABLENAME`
 bqExecuteDMLInsert=INSERT INTO `PROJECT_NAME.DATASET.TABLENAME` (Id, ProjectId, Dataset) values (100,'e2e-test','test automation')
+bqExecuteInsert=INSERT INTO `PROJECT_NAME.DATASET.TABLENAME` (ID,  Name, Price, TableName) values (1, 'string_1', 0.1, 'Test')
```
Comment (itsmekumari) [inline src/e2e-test/resources/pluginParameters.properties]: Insert some other values, as discussed yesterday.

**Review by itsmekumari — APPROVED** [review]
